### PR TITLE
Task login/register

### PR DESCRIPTION
**Pull request by nomorechokedboy**

### CODE REVIEW

File: src/containers/LoginForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { joiResolver, useForm } from "@mantine/form";
+import { CSSProperties, memo, useMemo } from "react";
+import { HiLockClosed, HiMail } from "react-icons/hi";
+import Title from "../../components/Title";
+import styles from "./styles.module.scss";
+import loginSchema, { LoginData } from "./validate";
+
+const LoginForm = memo(() => {
+    const { colorScheme } = useMantineColorScheme();
+    const { colors } = useMantineTheme();
```
Comment (cpea2506) [inline src/containers/LoginForm/index.tsx:29]: `{ colorScheme }` can be destructed from `useMantineTheme`

File: src/containers/LoginForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    const formStyle: CSSProperties = useMemo(
+        () =>
+            colorScheme === "dark"
+                ? { backgroundColor: colors[colorScheme][6] }
```
Comment (cpea2506) [inline src/containers/LoginForm/index.tsx:33]: `colors.dark[6]` can be used instead.

File: src/containers/Login/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    border-radius: 5px;
+    background-color: #fff;
+    padding: 20px;
+}
+.row {
+    margin-bottom: 20px;
+
+    &:last-child {
+        display: grid;
+        place-items: center;
```
Comment (cpea2506) [inline src/containers/Login/styles.module.scss]: we have `grid-center` mixin to center grid item

File: src/containers/LoginForm/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+.row {
+    margin-bottom: 20px;
+
+    &.heading {
+        @include flex-center-between;
+    }
+
+    &:last-child {
+        display: grid;
+        place-items: center;
```
Comment (cpea2506) [inline src/containers/LoginForm/styles.module.scss:23]: another `grid-center` mixin.

File: src/containers/LoginForm/index.tsx (hunk 2 above)
Comment (nomorechokedboy) [reply src/containers/LoginForm/index.tsx:33]: It is the same, we can skip this

File: src/containers/LoginForm/styles.module.scss (hunk 4 above)
Comment (nomorechokedboy) [reply src/containers/LoginForm/styles.module.scss]: I have changed it, please review my change

**Review by cpea2506 — APPROVED** [review]
LGTM

**Pull request merged**